src/app/forms/components/defect/defect.component.html

```diff
@@ -0,0 +1,50 @@
+<ng-container [ngSwitch]="edit">
+  <ng-container *ngSwitchCase="true">
+    <ng-temnplate [ngTemplateOutlet]="editDefect" [ngTemplateOutletContext]="{ data: defectData }"></ng-temnplate>
```

Comment:
typo?

Reply:
```suggestion
    <ng-template [ngTemplateOutlet]="editDefect" [ngTemplateOutletContext]="{ data: defectData }"></ng-template>
```

Not sure how suggestions work but hopefully it'll mean you can just click accept rather than having to set up a whole new commit for it!

Reply:
Good catch! Weirdly enough this doesn't break and works completely fine because it's been used with template outlet. Pushing fix.